src/main/webapp/app/exercises/quiz/manage/quiz-exercise-detail.component.html

```diff
@@ -425,5 +433,90 @@ <h5><span *ngIf="question.type === SHORT_ANSWER" class="badge badge-success alig
                 ></button>
             </div>
         </ng-template>
+        <ng-template #existingQuestionsFromExamTemplate>
+            <div class="form-group question-options existing-question-container">
+                <div class="existing-question-select-course w-100">
+                    <span jhiTranslate="artemisApp.quizExercise.selectExam" class="font-weight-bold align-middle"></span>
+                    <select class="form-control ml-5" [(ngModel)]="selectedExamId" (change)="onExamSelect()" style="flex: 1">
+                        <option *ngFor="let exam of exams" [value]="exam.id">{{ exam.title }}</option>
+                    </select>
+                </div>
+                <div class="existing-question-filter w-100" *ngIf="allExistingQuestions.length !== 0">
+                    <input class="form-control search-query" [(ngModel)]="searchQueryText" placeholder="Search Query" />
+                    <div class="checkboxes">
+                        <div class="form-check custom-control custom-checkbox">
+                            <input type="checkbox" id="cbDnDFilterExam" class="form-check-input custom-control-input" [(ngModel)]="dndFilterEnabled" />
+                            <label class="form-check-label custom-control-label" for="cbDnDFilterExam" jhiTranslate="artemisApp.quizExercise.filter.dnd"> </label>
+                        </div>
+                        <div class="form-check custom-control custom-checkbox">
+                            <input type="checkbox" id="cbMCFilterExam" class="form-check-input custom-control-input" [(ngModel)]="mcqFilterEnabled" />
+                            <label class="form-check-label custom-control-label" for="cbMCFilterExam" jhiTranslate="artemisApp.quizExercise.filter.mcq"> </label>
+                        </div>
+                        <div class="form-check custom-control custom-checkbox">
+                            <input type="checkbox" id="cbShortAnswerFilterExam" class="form-check-input custom-control-input" [(ngModel)]="shortAnswerFilterEnabled" />
+                            <label class="form-check-label custom-control-label" for="cbShortAnswerFilterExam" jhiTranslate="artemisApp.quizExercise.filter.sa"> </label>
+                        </div>
+                    </div>
+                    <button class="btn btn-outline-dark" (click)="applyFilter()" jhiTranslate="artemisApp.quizExercise.filter.applyFilter">eq</button>
```

Comment:
What does eq mean here?

Reply:
To be honest, I don't know. I just added it since we want to be consistent and we used this term in the CourseFilter as well

Reply:
Removed in 5a9153e